scripts/fiss_site_tidy.R

```diff
@@ -168,13 +172,10 @@ form_fiss_site <- bind_rows(
            dplyr::any_of(form_raw_names_site),
            # add the time to help put the puzzle together after)
            survey_date,
-           time,
-           comments_2)
+           time)
 ) %>%
-  # join the comments
-  dplyr::mutate(comments = case_when(
-    !is.na(comments_2) ~ paste0(comments, '. ', comments_2, '. ', time),
-    T ~ paste0(comments, '. ', time))) %>%
+  # add time to end of comments, there are no comments in the comments_2 column so no need to combine
+  dplyr::mutate(comments = paste0(comments, ' ', time)) %>%
```

Comment:
`comments = case_when(` though so removing makes the code less flexible with the same result